database/changesets/20200909_ExternalExomeReprocessing.xml

```diff
@@ -0,0 +1,31 @@
+<?xml version="1.1" encoding="UTF-8" standalone="no"?>
+<databaseChangeLog
+        xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+        xmlns:ext="http://www.liquibase.org/xml/ns/dbchangelog-ext"
+        xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog-ext
+                        http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-ext.xsd
+                        http://www.liquibase.org/xml/ns/dbchangelog
+                        http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.8.xsd">
+    <changeSet author="ehigham@broadinstitute.org"
+               dbms="postgresql"
+               id="20200909"
+               logicalFilePath="20200909_ExternalExomeReprocessing.xml"
+               runInTransaction="false">
+        <sql>
+            ALTER TYPE pipeline ADD VALUE 'ExternalExomeReprocessing'
+        </sql>
+        <sql>
+            ALTER TABLE workload ALTER COLUMN input DROP NOT NULL
```

Comment:
I don't understand this. What are we DROPping?
Do we want to allow `input` to be `null` now?
What about just `""` for _no common prefix_?

Reply:
This is dropping the non-null attribute of the `input` column.
I could change the default value to `''` but xx doesn't use "input" so I'd have to write custom code to remove it (`unnullify` won't do it anymore).
From our discussion it seems it's not a big problem for you but if you'd prefer I didn't make this nullable then let me know.